xml/http_servers.xml

```diff
@@ -1929,7 +1929,16 @@
   </fingerprint>
   <fingerprint pattern="^CUPS/((?:\d\.)+\d+)$">
     <description>Server for the CUPS web interface.</description>
-    <example>CUPS/1.1</example>
+    <example service.version="1.1">CUPS/1.1</example>
+    <param pos="0" name="service.vendor" value="Apple"/>
+    <param pos="0" name="service.product" value="CUPS"/>
+    <param pos="0" name="service.family" value="CUPS"/>
+    <param pos="1" name="service.version"/>
+    <param pos="0" name="service.cpe23" value="cpe:/a:apple:cups:{service.version}"/>
+  </fingerprint>
+  <fingerprint pattern="^CUPS\/((?:\d\.)+\d+)\s*IPP\/\d+\.\d+$">
```

Comment:
Now that you aren't capturing the IPP version at all, IMO you'd be better off just adding the IPP version optionally to the existing CUPS regex, since these are otherwise identical.